src/MSAL.PCL/Internal/Http/HttpClientFactory.cs

```diff
@@ -35,7 +35,17 @@ internal class HttpClientFactory
         private static HttpClient _client;
         private static readonly object LockObj = new object();
         public static bool ReturnHttpClientForMocks { set; get; }
+        public const long MaxResponseContentBufferSizeInBytes = 1024*1024;
 
+        private static HttpClient CreateHttpClient()
```

Comment:
You are right. My implementation was based on an assumption that creating an (empty) instance from a class is so cheap that we can afford those extra CPU cycles to achieve cleaner code. I'm fine with either way. :-)